rllib/algorithms/ppo/ppo_rl_module.py

```diff
@@ -40,7 +36,7 @@ def input_specs_inference(self) -> SpecDict:
 
     @override(RLModule)
     def output_specs_inference(self) -> SpecDict:
-        return SpecDict({SampleBatch.ACTION_DIST: Distribution})
+        return SpecDict({SampleBatch.ACTIONS: None})
```

Comment:
Both explore and inference forward methods now return actions directly, either sampled from distribution (explore) or deterministic ones (inference).

Reply:
wait, when did this become the case? Forward method should return the Action_dist_inputs where we can call dist.from_logits() on and then do sampling from the caller side of RLModules. See https://github.com/ray-project/ray/pull/35085/

Reply:
Yes, @ArturNiederfahrenhorst PR fixed all this. Merged it and factored all the new logic into this PR.